docs/howto/account-billing/Delete-account.md

```diff
@@ -0,0 +1,19 @@
+# Delete account
+
+If you unfortunately need to cancel or leave us for any reason, you can delete your account in our [{{gui}}](https://{{gui_domain}}).
+Log in and click on the Profile button at the top right.
+
+![Orange avatar person with white
+background](assets/picture-profile.png)
+
+You will now see the _Account settings_ page. Choose the _Settings_ tab, in which you can delete your account.
+
+![A Settings tab with a red box around it](assets/accountsettings.png)
+
+Scroll to the bottom until you see the image below and if you absolutely sure click on the red button _Delete Account_ and follow the procedure.
+
+![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)
+
+Your services will be shut off in 3 days and from now your services stop costing you money.
```

Comment:
Assuming all user assets will be wiped in 72 hours, I would write this as follows:

```suggestion
Any assets you may still have in {{brand}} will be wiped after 72 hours. Please keep in mind that during that time you will be charged as usual.
```

Reply:
No, the assets will only be suspended after 3 days. After 10 days we will start deleting servers and other stuff.

Shall I instead write 

Any assets you may still have in {{brand}} will be wiped after 10 days. Please keep in mind that during that time you will be charged as usual.

Reply:
Aha, so assets get wiped irrevocably after 10 and not after 3 days. Does this make any difference in the pricing structure? Is the customer getting charged as usual during this 10-day time period? If so, then I would suggest this change:

 ```suggestion
Any assets you may still have in {{brand}} will be suspended after 3 days, and wiped irrevocably 7 days after suspension. Please be aware that during these 10 days, you will be getting charged as usual.
```